### ISSUE TITLE
Reunión 2018-03-07

### ISSUE THREAD

**Issue by chucheria** [RLadiesMadrid/2017_CallOfData#6]
¡Hemos vendido las entradas de la primera tanda! 🎉👏

Vamos a sacar la segunda con el resto de módulos, os resumo aquí para tener una reunión más o menos rápida esta tarde.

- Link al blog: https://medium.com/call-of-data
- Link a la suscripción a la newsletter: https://t.co/69gKSCUDCm
- Módulo de ponentes.
- Módulo de jurados.
- Explicación del evento.
- Explicación del hackathon.
- Módulo de patrocinadores.
- Link al archivo de patrocinio: https://docs.google.com/document/d/1Xw2JvX_bA3OJ9FUn84XfFMnZK697nJjE8W9aIvmJzDI/edit?usp=sharing
- Link al C4P.

Los textos los vamos a hacer @mariamedp y yo, los pasaremos también por @nimbusaeta e @ineshf para que estén al tanto.

Para el C4P vamos a usar https://www.papercall.io/, tengo creado el evento, me faltan los textos.

Estamos pensando en hacer una traducción al inglés puesto que hay gente interesada desde el extranjero. Me dijo @oneeyedman que él tiene un sistema para cambiar la web de idioma 🤨así que a la hora de maquetar hablad con él. 

Sé que es mucho trabajo pero ya es lo gordo y lo siguiente ya sería seguir añadiendo textos. Muchas gracias a todos!

**Comment by chucheria** [comment]
- Módulo de agenda <- futuro

**Comment by chucheria** [comment]
Me acabo de acordar de dos links más:

- Slack del CoD: https://callofdata-slack.herokuapp.com/
- Formulario para que se apunten los equipos.

**Comment by chucheria** [comment]
No estoy muy segura si tendremos un FAQ más adelante, pero por si acaso lo dejo aquí apuntado.

**Comment by nimbusaeta** [comment]
> Los textos los vamos a hacer @mariamedp y yo, los pasaremos también por @nimbusaeta e @ineshf para que estén al tanto

¿Tenemos los textos del año pasado? Mucho será igual

**Comment by chucheria** [comment]
@nimbusaeta sí, se los pasé a @mariamedp en el email para hacer los textos.
